docker/integration.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh -xe
+if [ -z $PRODUCT ]
+then
+  echo "PRODUCT env var for integration testing not set!"
+  exit 1
+fi
+
+rake build[$PRODUCT] &&
+zypper --non-interactive --no-gpg-checks in /oscbuild/$PRODUCT-x86_64/home/abuild/rpmbuild/RPMS/x86_64/* &&
+cucumber /tmp/connect/features/activation.feature /tmp/connect/features/activation_errors.feature
```

Comment:
at least help, localization and version.feature should work, to make it working completely we should add a card to the backlog.